code/datums/traits/negative.dm

```diff
@@ -426,11 +426,8 @@
 		if(H.client)
 			nearby_people++
 	var/mob/living/carbon/human/H = quirk_holder
-	if(prob(2 + nearby_people))
+	if(prob(2 + 2^nearby_people))
 		H.stuttering = max(3, H.stuttering)
-	else if(prob(min(3, nearby_people)) && !H.silent)
-		to_chat(H, "<span class='danger'>You retreat into yourself. You <i>really</i> don't feel up to talking.</span>")
-		H.silent = max(10, H.silent)
```

Comment:
I think slightly nerfing the max silence would be a better change than outright removing silence.

Reply:
I thought of that, but I didn't for two reasons.

A: Silence is definitely one of the most crippling downsides you can have.
B: I'm honestly not sure how to do that.